### DISCUSSION TITLE
Error during column datatype change [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2312] [Discord Threads]
> 🙋‍♂️ **saki**  
##   
![image0](https://cdn.discordapp.com/attachments/661905455894888493/981509084710834206/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### You had a column with type 'SingleLineText'. It had numbers stored in its rows. You get an error when you try to edit its datatype to 'Number'.    
####    
#### Is this observation correct?  
#  
> 👨‍🦲 **dstala**  
#### I could see this error. Looks like the empty cell is causing problem (may be is being treated as a string).    
####    
#### I was able to edit column from text > number of clearing the contents of empty cell explicitly. (right click on the cell - click clear data).  
#  
> 🙋‍♂️ **saki**  
#### yes, this was the case. the type is single line text, but even when i try to change it to Long Text I get the same error.  
![image0](https://cdn.discordapp.com/attachments/981509085298053180/981537789969571930/unknown.png)  
#  
> 🙋‍♂️ **saki**  
#### so this is a bug, right?  
#  
> 👨‍🦲 **dstala**  
#### yes  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981509085298053180)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
